mapie/classification.py

```diff
@@ -104,7 +109,7 @@ class MapieClassifier(BaseEstimator, ClassifierMixin):
         for evaluation quantiles and prediction sets in cumulated_score.
         Pass an int for reproducible output across multiple function calls.
 
-        By default ```1``.
+        By default ```None``.
```

Comment:
why changin 1 to None ?
mission ```

Reply:
I have fixed ``` for classification and regression.

Reply:
Why changing 1 to None? Because by default, we use `None` (see line 194).